### PR TITLE
Replace docs example with actual code

### PR DESCRIPTION
**Pull request by cijothomas**
Want to get some feedback on this approach. Moving sample code to actual project will ensure they always remain upto date.

Please provide a brief description of the changes here. Update the
`CHANGELOG.md` for non-trivial changes.

For significant contributions please make sure you have completed the following items:

* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: docs/trace/getting-started/README.md [hunk 1]
```diff
@@ -25,45 +25,22 @@ package:
...
-
-        using (var activity = MyActivitySource.StartActivity("SayHello"))
-        {
-            activity?.SetTag("foo", 1);
-            activity?.SetTag("bar", "Hello, World!");
-        }
-    }
-}
-```
+Update the `Program.cs` file with the code from Program.cs:
```
Comment (cijothomas) [inline docs/trace/getting-started/README.md:28]: should we instead modify this entire instructions to:
run this project using `dotnet run`.

And explanation of what this program does.

?

Comment (alanwest) [reply docs/trace/getting-started/README.md:28]: Definitely makes sense to me to not have code spread throughout markdown files. It's bound to get out of sync. You also plan to get rid of the sample output? That'll also get out sync.

File: docs/trace/getting-started/README.md [hunk 2]
```diff
@@ -75,7 +52,7 @@ Tracer](https://github.com/open-telemetry/opentelemetry-specification/blob/maste
 The activitysource instance is used to start an `Activity` which represent
```
Comment (alanwest) [inline docs/trace/getting-started/README.md:52]: I think it makes sense to link `ActivitySource` and `Activity` to the Microsoft docs just to clearly draw people to the relationship.

File: docs/trace/getting-started/README.md (hunk 1 above)
Comment (cijothomas) [reply docs/trace/getting-started/README.md:28]: For the getting started in 5 minutes one, I am okay to keep the sample output. Atleast it helps a totally new user to confirm what he see when he runs the sample is the expected behaviour..

File: docs/trace/getting-started/README.md [hunk 3]
```diff
@@ -75,7 +52,7 @@ Tracer](https://github.com/open-telemetry/opentelemetry-specification/blob/maste
...
 [OpenTelemetry
 Span](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#span).
-`Sdk.CreateTracerProvider` sets up the OpenTelemetry Sdk, and
-configures it to subscribe to the activities from the source
+OpenTelemetry TracerProvider which is setup at the beginning is
+configured to subscribe to the activities from the source
```
Comment (alanwest) [inline docs/trace/getting-started/README.md:56]: Maybe it's just me, but this reads kinda funny. How about: "An OpenTelemetry TracerProvider is configured to subscribe to the activities from the source..."?

File: docs/trace/getting-started/README.md (hunk 1 above)
Comment (reyang) [reply docs/trace/getting-started/README.md:28]: These are the changes I think we will need to make before GA:
1. Move the `*.md` files to individual folder, for example `docs/trace/getting-started/README.md`.
2. Document has to reflect the latest code. Ideally we can make this automated, or at least have a CI check.
3. Have a way to change the project reference to package reference.

Comment (reyang) [inline docs/trace/getting-started/README.md:28]:
```suggestion
Update the `Program.cs` file with the code from [Program.cs](./Program.cs):
```

File: docs/trace/getting-started/README.md (hunk 3 above)
Comment (cijothomas) [reply docs/trace/getting-started/README.md:56]: :D agree. Let me rephrase.

File: docs/trace/getting-started/README.md (hunk 2 above)
Comment (cijothomas) [reply docs/trace/getting-started/README.md:52]: I did not do it intentionally as the official microsoft docs don't yet exist. The link to ActivitySource in API doc also gives the wrong impression that its only available for .NET 5 only.
https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.activitysource?view=net-5.0

File: README.md [hunk 4]
```diff
@@ -12,8 +12,7 @@ The .NET [OpenTelemetry](https://opentelemetry.io/) client.
 
 ## Getting Started
 
-If you are new here, please [get started in 5 minutes](./docs/trace/getting-started.md).
-
+If you are new here, please [get started in 5 minutes](./docs/trace/getting-started/README.md).
```
Comment (CodeBlanch) [inline README.md:15]: Seems like something is missing in that sentence? Either that or we're asking them to wait 5 minutes before getting started 🤣 

How about:

`If you are new here, please see [get started in 5 minutes](./docs/trace/getting-started/README.md).`

or just

`If you are new here, see [get started in 5 minutes](./docs/trace/getting-started/README.md).`

File: docs/trace/getting-started/README.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    foo: 1 bar: Hello, World! baz: [1, 2, 3]
+```
+
+Congratulations! You are now collecting traces using OpenTelemetry.
+
+What does the above program do?
+
+The program creates an `ActivitySource` which represents [OpenTelemetry
+Tracer](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#tracer).
+The activitysource instance is used to start an `Activity` which represent
```
Comment (CodeBlanch) [inline docs/trace/getting-started/README.md:47]: nti: "represents"

"activitysource" is that the name of a variable in the program? Otherwise should probably use "ActivitySource".

Comment (reyang) [reply docs/trace/getting-started/README.md:47]: #1061.

File: README.md (hunk 4 above)
Comment (reyang) [reply README.md:15]: #1061.
